src/uikit.cs

```diff
@@ -4872,10 +4862,33 @@ interface UIColor : NSSecureCoding, NSCopying
 		bool GetRGBA2 (out nfloat red, out nfloat green, out nfloat blue, out nfloat alpha);
 #endif
 
+		// From the NSItemProviderReading protocol, a static method.
+		[Static]
+		[iOS (11,0), NoWatch, NoTV]
+		[Export ("readableTypeIdentifiersForItemProvider", ArgumentSemantic.Copy)]
+#if !WATCH && !TVOS
```

Comment:
needed ? there's a [NoWatch] and [NoTV] just above

Reply:
It avoids a compiler warning when compiling the api definition (the protocol is only inlined for !WATCH and !TVOS, which means that the `new` keyword is only needed for !WATCH and !TVOS, and if the `new` keyword is needlessly present, the C# compiler complains.